test/QtDBMigrationTest.cpp

```diff
@@ -23,7 +23,7 @@
 #include <QString>
 #include <QtTest>
 
-#include "QtDBMigration.h"
+#include "../src/QtDBMigration.h"
```

Comment:
This means something was wrong with your build configuration

Reply:
Should be fixed.